src/app/vault/groupings.component.html

```diff
@@ -32,44 +32,19 @@ <h2>{{'types' | i18n}}</h2>
-                            <path fill="#FF9300" d="M12 11.5c1.5 0 3 2.25 3 3s0 1.5-.75 1.5H3.75C3 16 3 15.25 3 14.5s1.5-3 3-3 .75.75 3 .75 1.5-.75 3-.75zM28 14v1H17v-1h11zm0-3v1H17v-1h11zM9 4c1.657 0 3 1.511 3 3.375s-1.343 3.375-3 3.375-3-1.511-3-3.375S7.343 4 9 4zm19 4v1H17V8h11zm-5-3v1h-6V5h6z"/>
-                        </g>
-                    </svg>
+                    <svg class="icon-type" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><g filter="url(#filter0_d_3096_936)"><path d="M15.992 5.874c5.857 0 10.606 4.102 10.606 9.16 0 5.06-4.749 9.16-10.606 9.16a12.12 12.12 0 0 1-3.432-.49c-1.203.936-3.069 1.722-5.596 2.356a.485.485 0 0 1-.541-.706l.18-.329c.84-1.544 1.364-2.811 1.572-3.8-1.733-1.63-2.79-3.804-2.79-6.19 0-5.06 4.749-9.16 10.607-9.16Z" fill="#fff"/></g><g clip-path="url(#clip0_3096_936)"><rect width="32" height="32" rx="10" fill="url(#paint0_linear_3096_936)"/><g filter="url(#filter1_d_3096_936)" fill="#fff"><path d="M15.99 16.016a4.5 4.5 0 1 0 0-9 4.5 4.5 0 0 0 0 9Zm8.85 6.028c-.68-1.907-2.795-4.028-8.85-4.028s-8.171 2.122-8.85 4.029c-.599 1.682 1.081 2.971 2.867 2.971h11.965c1.786 0 3.466-1.289 2.867-2.971Z"/></g></g><defs><filter id="filter0_d_3096_936" x="3.753" y="4.569" width="24.477" height="23.463" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy=".326"/><feGaussianBlur stdDeviation=".816"/><feComposite in2="hardAlpha" operator="out"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_3096_936"/><feBlend in="SourceGraphic" in2="effect1_dropShadow_3096_936" result="shape"/></filter><filter id="filter1_d_3096_936" x="4.16" y="4.731" width="23.66" height="23.714" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy=".571"/><feGaussianBlur stdDeviation="1.429"/><feComposite in2="hardAlpha" operator="out"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_3096_936"/><feBlend in="SourceGraphic" in2="effect1_dropShadow_3096_936" result="shape"/></filter><linearGradient id="paint0_linear_3096_936" x1="5.548" y1="29.127" x2="30.188" y2="1.958" gradientUnits="userSpaceOnUse"><stop offset=".23" stop-color="#F74EA9"/><stop offset="1" stop-color="#FFB2A8"/></linearGradient><clipPath id="clip0_3096_936"><path fill="#fff" d="M0 0h31.983v31.983H0z"/></clipPath></defs></svg>
```

Comment:
I did not run the app, but by previewing the SVGs those icons seems to be pretty heterogenous, aren't they?

Also in the icon.component.html file the note icon is still the old one. Is that expected?

Reply:
They all looks good thanks to the .icon-type that manage the size. For the moment I have no new icons for notes.